main/create_simulated_body_model.py

```
@@ -266,6 +266,20 @@ def _setup_ui(self) -> None:
         right.addLayout(vis_row)
 
         main_layout.addLayout(right, stretch=1)
+        # Initialize common zoom/pan controller for left-button pan and wheel zoom
+        self._zoom_ctl: Optional['ZoomPanController'] = None
```

Ruff (0.14.5)

Remove quotes from type annotation

Remove quotes

(UP037)